test/mainnet/helpers/SetupVault.sol

```diff
@@ -24,7 +28,7 @@ contract SetupVault is Test, MainnetActors, Etches, BaseRules {
 
         bytes4 selector = bytes4(keccak256("upgradeAndCall(address,address,bytes)"));
 
-        bytes memory initData = abi.encodeWithSelector(ynETHxVault.initializeV2.selector, 18, 0);
+        bytes memory initData = abi.encodeWithSelector(YnETHxVault.initializeV2.selector, 18, 0);
```

Comment:
i don't think we'll be able to use this script as something that sends to the timelock we'd just load up the data manually